helm/nessie/README.md

```diff
@@ -83,6 +83,35 @@ aws_secret_access_key=YOURSECRETKEYDATA
 * Create K8s Namespace: `kubectl create namespace nessie-ns`
 * Install Nessie Helm chart: `helm install nessie -n nessie-ns helm/nessie`
 
+### Ingress with Minikube
+
+This is broadly following the example from https://kubernetes.io/docs/tasks/access-application-cluster/ingress-minikube/
+
+* Start Minikube cluster: `minikube start`
+* Enable NGINX Ingress controller: `minikube addons enable ingress`
+* Verify Ingress controller is running: `kubectl get pods -n ingress-nginx`
+* Configure Nessie with the following ingress settings
```

Comment:
Where / how would a user do this?

Reply:
in the `values.yaml` of the helm chart

Reply:
I'll make this more clear in the next PR